code-challenges.test.js

```diff
@@ -1,5 +1,7 @@
 // ASSESSMENT 3: Coding Practical Questions with Jest
 
+// SIDENOTE : i am done for the weekends work. This HW has bit to the pulp. I didn't get to the jest because i have not figured out the working function for it to pass. I think i need some time from it. I do have a quastion. How long is too long for doing a challenge? I mean the reason i am not asking for help is because these are fundamentals and i think if i want to understand them where else would i do it. Assessment is the best way to do it. However, i do think it is not enough time. I am not done. I will be working on it through out the week. However, i will have to return what i have. 
```

Comment:
It's okay if you don't have the correct answer when you submit it, all that matters is that you gave it a solid try, which I can see that you always do. And just so you know, I think this is a fantastic assessment!